models/ssh_key.go

```diff
@@ -543,16 +545,38 @@ func RewriteAllPublicKeys() error {
 		_, err = f.WriteString((bean.(*PublicKey)).AuthorizedString())
 		return err
 	})
-	f.Close()
+
 	if err != nil {
+		f.Close()
```

Comment:
`defer f.Close()` is good practice, if I'm not familiar with the code-base I'd assume that it defers it and then we have a bug 🙁